browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1479,7 +1479,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		value = parseFloat(data.value);
 
 		// time formatter or empty field
-		if (data.unit === ':' || (!data.unit && !data.text)) {
+		if (data.hasTimeFormatter) {
```

Comment:
did you find any field which uses this condition?

I looked again and it seems widget with `"type"="time"` uses `JSDialog.timeField`, and it's the only widget which can trigger this condition... so probably it's leftover code from before we introduced `JSDialog.timeField`

Reply:
File -> Properties -> Custom properties with field type date time has it it seems